src/scala/scripts/PBGenerateRequest.scala

```diff
@@ -2,48 +2,38 @@ package scripts
 
 import java.nio.file.{Files, Path, Paths}
 
-class PBGenerateRequest(val jarOutput: String, val scalaPBOutput: Path, val scalaPBArgs: List[String], val protoc: Path)
+class PBGenerateRequest(val jarOutput: String, val scalaPBOutput: Path, val scalaPBArgs: List[String], val includedProto: List[(String, String)], val protoc: Path)
```

Comment:
wonder why this isn't a case class and we can remove all the `val`s...

Reply:
will try make that change to make it shorter